lib/smart_answer_flows/state-pension-topup/questions/how_much_extra_per_week.govspeak.erb

```diff
@@ -6,6 +6,10 @@
   Enter a number between 1 and 25.
 <% end %>
 
-<% content_for :error_message do %>
-  Enter an amount in whole pounds between 1 and 25.
+<% content_for :error_not_whole_number do %>
+  Enter an amount which is a whole number.
```

Comment:
I wonder whether this should still use "whole pounds", i.e. "Enter an amount in whole pounds"?